src/main/java/org/apache/pulsar/manager/controller/UsersController.java

```diff
@@ -218,6 +218,11 @@ public ResponseEntity<Map<String, Object>> createSuperUser(@RequestBody UserInfo
             result.put("error", "Please provider password");
             return ResponseEntity.ok(result);
         }
+        Optional<UserInfoEntity> optionalUserEntity =  usersRepository.findByUserName(userInfoEntity.getName());
+        if (optionalUserEntity.isPresent()) {
+            result.put("error", "Superuser already exists");
+            return ResponseEntity.ok(result);
```

Comment:
The previous solution of using ok for errors is strange, but I guess it's fine to follow the original bad convention.

Reply:
Yes thanks for pointing it out, there's some other ❓ , but I'm trying to focus on my use-case here following as you say same convention, not really sure what else might depend on the original convention in terms of client handling